doc(release): Release notes for web 22.10.23 / gorgone 22.10.8 (#3405)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md

## Centreon Web

### 22.10.23

Release date: `June 26, 2024`

#### Bug fixes

- [Configuration] Fixed mass change on services.

> - [Unattended] Fixed unattended not working on Oracle Linux distributions.

Reviewer: ```suggestion
- [Unattended] The unattended script now works on Oracle Linux distributions.
```
